tests/span_tests.cpp

```diff
@@ -800,10 +702,9 @@ TEST(span_test, first)
     {
         span<int, 5> av = arr;
 #ifdef CONFIRM_COMPILATION_ERRORS
-        EXPECT_TRUE(av.first<6>().size() == 6);
-        EXPECT_TRUE(av.first<-1>().size() == -1);
```

Comment:
Why did `first<-1>` disappear?

Reply:
-1 is an `int`, the template parameter is a `std::size_t`. clang unit tests fail because of a warning for this narrowing conversion. I could have written `av.first<static_cast<std::size_t>(-1)>()`, but I thought that this does not add to the test coverage. If you want that test, I can reintroduce it.

Reply:
Ah! I missed that. I agree with you removing it.